Cargo.toml

```diff
@@ -169,6 +169,7 @@ pallet-evm-precompile-simple = { version = "2.0.0-dev", git = "https://github.co
 pallet-hotfix-sufficients = { version = "1.0.0", git = "https://github.com/AstarNetwork/frontier.git", branch = "polkadot-v0.9.39", default-features = false }
 
 # Golden Gate
+account-filter = { package = "substrate-account-filter", git = "https://github.com/GoldenGateGGX/ggx-frames.git", branch = "decentrilized-validator-manager", default-features = false }
```

Comment:
We can use `main` branch when https://github.com/GoldenGateGGX/ggx-frames/pull/2 will be merged.